packages/i18n/lib/i18n-dev.ts

```diff
@@ -0,0 +1,26 @@
+import { DevLocale, MessageKey } from './type';
+import { defaultLocale, getMessageFromLocale } from './getMessageFromLocale';
+
+function translate(key: MessageKey, substitutions?: string | string[]) {
+  const message = getMessageFromLocale(t.devLocale)[key].message;
+
+  if (!substitutions) {
+    return message;
+  }
+
+  // TODO: Implement all substitution logic (placeholders fields)
+  if (Array.isArray(substitutions)) {
+    return substitutions.reduce((acc, cur, idx) => acc.replace(`$${idx + 1}`, cur), message);
+  }
+  return message.replace(/\$(\d+)/, substitutions);
+}
+
+function removePlaceholder(message: string) {
+  return message.replace(/\$\d+/g, '');
+}
+
+export const t = (...args: Parameters<typeof translate>) => {
+  return removePlaceholder(translate(...args));
```

Comment:
I looking for that, and my thoughts about it have destroyed xD

I think substitusion is placeholders.

PLS create more examples and explanation in README 🔥